keel-artifact/src/test/kotlin/com/netflix/spinnaker/keel/artifacts/DebianArtifactSupplierTests.kt

```diff
@@ -52,6 +52,14 @@ internal class DebianArtifactSupplierTests : JUnit5Minutests {
       metadata = mapOf("releaseStatus" to SNAPSHOT, "buildNumber" to "1", "commitId" to "a15p0")
     )
 
+    val actualVersionInPublishedArtifact = PublishedArtifact(
```

Comment:
If this is the actual format of a Debian published artifact (i.e. without the name part in the `version` string), then I think a better approach would be to modify `latestArtifact` above and just keep the existing test.

Reply:
yeah good point.